client/src/components/manageProjects/editProject.js

```diff
@@ -1,207 +1,237 @@
-import React, { useState }  from 'react';
+import React from 'react';
 import '../../sass/ManageProjects.scss';
 import EditableField from './editableField';
-import { REACT_APP_CUSTOM_REQUEST_HEADER } from "../../utils/globalSettings";
+import { REACT_APP_CUSTOM_REQUEST_HEADER } from '../../utils/globalSettings';
 
+// Need to hold user state to check which type of user they are and conditionally render editing fields in this component
+//for user level block access to all except for the ones checked
 
-const EditProjectInfo  = ( props ) => {
-
+const EditProject = (props) => {
```

Comment:
Were you an English major?  I love that the name of this component bothered you enough to change it here, and in all of the other places that it appears.  Miriam Webster would be proud of you.

Reply:
😆 I can barely put two words of English together! But yeah, I find it confusing when the component and the file don't have the same name.